src/cloud_logs/stat.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Cloud logs stats."""
+
+from typing import Any, Dict, List
+
+from openrelik_worker_common.reporting import MarkdownDocument
+from openrelik_worker_common.reporting import MarkdownDocumentSection
+from openrelik_worker_common.reporting import MarkdownTable
+
+
+class GoogleCloudLogStat:
+    """Class for tracking Google Cloud audit log stats."""
+
+    def __init__(self, log_source: str) -> None:
+        """Initializes GoogleCloudLogStat."""
+        self.log_source = log_source
+        self.skipped_log_count: int = 0
+        self.payload_type_stat: Dict[str, int] = {}
+        self.service_stat: Dict[str, int] = {}
+        self.method_stat: Dict[str, int] = {}
+        self.principal_email_stat: Dict[str, int] = {}
+
+    def UpdateCloudLogStat(self, log_entry: Dict[str, Any]) -> None:
```

Comment:
Please use snake_case to stay consistent with the rest of the code base.

`update_cloud_log_stat`

Reply:
Sure. This was ported from Plaso, so it had the Plaso style.